_search-plugins/star-tree-index.md

```diff
@@ -140,14 +140,18 @@ Star-tree indexes can be used to optimize queries and aggregations.
 
 ### Supported queries
 
-The following queries are supported as of OpenSearch 2.18:
+The following queries are supported as of OpenSearch 2.19:
 
 - [Term query]({{site.url}}{{site.baseurl}}/query-dsl/term/term/)
+- [Terms query]({{site.url}}{{site.baseurl}}/query-dsl/term/terms/)
 - [Match all docs query]({{site.url}}{{site.baseurl}}/query-dsl/match-all/)
+- [Range query]({{site.url}}{{site.baseurl}}/query-dsl/term/range/)
```

Comment:
We don't support `queryRelation` as it mostly used for Date OR [NumericRange field types](https://opensearch.org/docs/latest/field-types/supported-field-types/range/)  

We can capture the same here.

Reply:
@sandeshkr419: Has this comment been addressed?

Reply:
Unsupported `queryRelation` operation is applicable only on `range` type fields which we do not support via star-tree hence we do not need to include this detail.